src/jc_test.h

```diff
@@ -976,13 +980,35 @@ template<template <typename T> class BaseClass> struct jc_test_template_sel {
                 JC_TEST_MAKE_NAME2(testfixture,Types)>::register_test(#testfixture, #testfn, 0);            \
     template<typename T> void JC_TEST_MAKE_CLASS_NAME(testfixture,testfn)<T>::TestBody()
 
+#define JC_TEST_CAT_IMPL(s1, s2) s1##s2
+#define JC_TEST_CAT(s1, s2) JC_TEST_CAT_IMPL(s1, s2)
+#ifdef __COUNTER__
```

Comment:
Interesting, I've never even heard of `__COUNTER__` before. I'm curious, what's the actual benefit over using `__LINE__`?
I've never used a compiler that doesn't support `__LINE__`? :)

Reply:
I agree with minimalism.